debian/rules

```diff
@@ -61,12 +61,17 @@ DEPENDS += ansible, \
 DEPENDS += ubuntu-dbgsym-keyring,
 
 #
-# This package contains a consolidation for third-party packages built
-# by delphix as well as build info for all packages built by the linux-pkg
-# framework.
+# This package provides the build information for all packages built by
+# the linux-pkg framework.
 #
 DEPENDS += delphix-extra,
 
+#
+# The following packages provide additional set of packages that are
+# common part of the Delphix platform.
+#
+DEPENDS += td-agent,
```

Comment:
why not put this in the list of packages above? the “minimal” set of packages..

related, why have it be a dependency of the platform package rather than the virtualization package?

Reply:
@prakashsurya I didn't put it in the "minimal" set because per its comments, its for packages "necessary to create a working system" and this td-agent while wanted isn't essential for a working system

Regarding second comment, while this package is used by the virtualization/appstack, it is not limited to that - other components of delphix platform like masking, orbital integration etc will eventually use it.
And logically this (td-agent/fluentd) forms an essential piece of the delphix platform for any subsystem to push out Insight/diagnostic data out of DDP.

Reply:
Sorry, now it makes sense. I thought the comment said something about the list creating the "minimal platform"; I didn't actually re-read it last night.

I'm OK creating a new "section" like you did here. I'd vote for pulling `delphix-extra` into this new section as well (or not, up to you).

> other components of delphix platform like masking, orbital integration etc will eventually use it.

These other components could also depend on the td-agent.. e.g. masking and virtualization packages could both depend on td-agent; but that's a little redundant since masking already depends on td-agent. Same for any orbital component; presumably that'll have a package that could depend on td-agent in the future as well.

I'm not opposed to adding the dependency here, it's just the argument for doing it here vs. the actual consumers of the package feels is a little weak (to me, at least).